datalad/consts.py

```diff
@@ -28,6 +29,9 @@
 
 DATASETS_TOPURL = "http://datasets.datalad.org/"
 
+# Centralized deployment
+LOCAL_CENTRAL_PATH = join(expanduser('~'), 'datalad')
```

Comment:
AppDirs settings primarily seems to point to locations which user never interacts with directly by navigating into those dirs, e.g. the one which would be appropriate is 
```ipython
*In [4]: appdirs.user_data_dir('datalad')
Out[4]: '/home/yoh/.local/share/datalad'
```
but that is not the location I would expect anyone to remember or go to to explore.  That is why it is under user's home : `expanduser('~`')` which afaik should work also on windows
/home/yoh/.local/share/datalad